src/components/game/Chat/Chat.js

```diff
@@ -43,7 +43,7 @@ export const Chat = ({ messages, onSend, onClose }) => {
   return (
     <Segment.Group className={styles.chat}>
       <Segment compact className={styles.header}>
-        Chat
+        Game chat
```

Comment:
Can we add this to translations?

Reply:
sure!

Reply:
@glowacki-dev any idea for good polish translation? _"Czat gry"_ sounds weird

Reply:
Maybe just "Czat"? Or something like "Wiadomości" or "Rozmowa"?

Reply:
I'll go with just "Czat" 👍